schema/update-action.schema.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "UpdateActionMessage",
+  "description": "A template for messages that execute predetermined actions.",
+  "version": "1.0.0",
+  "type": "object",
+  "properties": {
+    "content": {
+      "type": "object",
+      "properties": {
+        "bucket_id": {
+          "type": "string",
+          "description": "A bucket identifier for the addon. This is used in order to anonymize telemetry for history-sensitive targeting."
+        },
+        "action": {
+          "type": "object",
+          "description": "Additional data provided as argument when executing the action",
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "data": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "URL data to be used as argument to the action"
+                },
+                "expireDelta": {
+                  "type": "number",
+                  "description": "Expiration timestamp to be used as argument to the action"
+                },
+                "expire": {
+                  "type": "number",
+                  "description": "Timestamp in ms when the message should expire" 
+                }
+              },
+              "required": ["url"],
+              "additionalProperties": false
+            }
+          },
+          "required": ["id"],
+          "additionalProperties": false,
+          "description": "Optional action to take in addition to showing the notification"
+        }
+      },
+      "additionalProperties": false,
+      "required": ["action"]
```

Comment:
We still need `bucket_id` to be required until we drop it for Moments telemetry.